koji/vars/external_repos.yml

```diff
@@ -20,17 +20,17 @@ external_repos:
     url: http://mirror.centos.org/centos/7/updates/x86_64/
   # EL8 repositories
   - name: centos8-baseos
-    url: http://mirror.centos.org/centos/8/BaseOS/$arch/os/
+    url: http://vault.centos.org/centos/8/BaseOS/$arch/os/
   - name: centos8-appstream
-    url: http://mirror.centos.org/centos/8/AppStream/$arch/os/
+    url: http://vault.centos.org/centos/8/AppStream/$arch/os/
   - name: test-flat-el8
     url: http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001/
   - name: centos8-devel
     url: https://vault.centos.org/8.4.2105/Devel/$arch/os/
   - name: centos8-configmanagement-ansible-29
-    url: http://mirror.centos.org/centos/8/configmanagement/$arch/ansible-29/
```

Comment:
It looks like the SIG isn't archived. Perhaps they asked to be rebuilt against RHEL and kept? (https://lists.centos.org/pipermail/centos-devel/2022-January/120120.html said that's an option).

Reply:
For the time being, this directory exists on vault:

https://vault.centos.org/centos/8/configmanagement/x86_64/ansible-29/